include/tvc.h

```diff
@@ -85,4 +86,12 @@ extern int __LIB__ getk();
 #define tvc_clrscr asm_tvc_cls
 extern void __LIB__ asm_tvc_cls(void);
 
+
+/**
+ * Sets the video mode of TVC (2 colours, 4 colours, 16 colours), clears screen
+ * and initialize cursor positions.
+ */
+#define tvc_vmode asm_tvc_vmode
```

Comment:
We tend to avoid having asm_XXXX() functions in the header file - functions named like that are for use by assembler programmers since they have register based input parameters.

When they are the same we create a file in the library that does something like this:

```
PUBLIC tvc_vmode
EXTERN asm_tvc_vmode

defc tvc_vmode = asm_tvc_vmode
```

Reply:
Several cases I didn't understand the pattern. Thanks for the guides I will fix these issues soon.